spec/helpers/util.js

```diff
@@ -60,6 +60,14 @@ function isIE() {
     return ((navigator.appName === 'Microsoft Internet Explorer') || ((navigator.appName === 'Netscape') && (new RegExp('Trident/.*rv:([0-9]{1,}[.0-9]{0,})').exec(navigator.userAgent) !== null)));
 }
 
+function isEdge() {
```

Comment:
yeah, it can be.  I was trying to follow the naming convention, and had this return `0` if it wasn't edge, so that calls to `isEdge()` would work as you would expect, but the bonus is that you can also find out the exact version too.

If you think we should, I'm fine with renaming it.